src/app/app.component.ts

```diff
@@ -118,13 +143,22 @@ export class AppComponent implements OnInit {
     const request = new CommandRequest(userCommand);
 
     this.commandService.sendCommand(request).pipe(
-      mergeMap(commandId => timer(0, this.COMMAND_STATUS_CHECK_INTERVAL)
+      mergeMap(commandId => {
```

Comment:
rethink this one, please. the server should return `error` status and then you'll just need to add `error` case down below